### ISSUE TITLE
Integrae OERs into Education Materials Page

### ISSUE THREAD

**Issue by jstet** [CorrelAid/correlaid_website#210]
OER and Workshops (see directus) will be merged in the future, but for first release we will only display workshops

**Comment by jstet** [comment]
Carousel Cards for e.g. insta content

**Comment by jstet** [comment]
@ZoeWolter is this still something you would like to be implemented?

**Comment by ZoeWolter** [comment]
@jstet Yep, especially as we're going to start a Data Literacy Course Social Media Series - would it be possible to have this one on the page of the "Daten verstehen und nutzen" Course as well?
